yaksh/decorators.py

```diff
@@ -1,4 +1,4 @@
-from django.shortcuts import render_to_response, redirect
+from django.shortcuts import render, render_to_response, redirect
```

Comment:
```render_to_response``` is replaced by ```render```, then ```render_to_response``` can be removed from the import as it is not used any more.